.github/workflows/nodejs.yml

```diff
@@ -1,6 +1,15 @@
 name: Node CI
 
-on: [push]
+on:
```

Comment:
Let's change line 3 - 12 to below 

```
on:
  - push
  - pull_request
```

similar to this https://github.com/gh-conf/rhof/blob/master/.github/workflows/nodejs.yml#L3-L5

Reply:
The requested changes has been done. **@arshadkazmi42**